src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/SmsSecondFactor/index.tsx

```diff
@@ -129,66 +138,81 @@ export const SmsSecondFactor: React.FC<SmsSecondFactorProps> = ({
     })
   }
 
+  const show2FAWarning = isArtsyEmail(me.email!)
+
   return (
     <>
-      <Flex
-        p={2}
-        border="1px solid"
-        borderColor="black10"
-        flexDirection={["column", "row"]}
-      >
-        <Box flexBasis="50%">
-          <Text variant="lg-display">Use Text Messages</Text>
-
-          {enabledSecondFactorLabel && (
-            <Text variant="lg-display" color="black60">
-              {enabledSecondFactorLabel}
+      <Flex flexDirection="column" border="1px solid" borderColor="black10">
+        {show2FAWarning && (
+          <Message variant="warning">
+            <Text>
+              Artsy employees are encouraged to use MFA applications instead of
```

Comment:
**suggestion**: To match the copy used on this page, what do you think of something like this?

> Artsy employees are encouraged to use the "App Authenticator" 2FA method via 1Password (or your preferred password manager). You may find a detailed walkthrough [here in Notion](https://www.notion.so/artsy/Guide-Using-1Password-2FA-with-Artsy-net-2c34243df67e4db3915e84c2ad4636b0). 

Noticing that the documentation linked to via https://artsy.net/employees-mfa-instructions heavily features the SMS enrollment method over app authenticator.